### PR TITLE
Clean test warnings

### PR DESCRIPTION
**Pull request by a-peltier**
Fix warnings as discussed in #1328 

The only remaining warnings come from react-motion which still uses the deprecated lifecycle methods but the MR to fix these is still pending, see https://github.com/chenglou/react-motion/issues/569

### CODE REVIEW

File: packages/react-vis/tests/components/line-series-canvas.test.js [hunk 1]
```diff
@@ -11,8 +11,9 @@ describe('LineSeriesCanvas', () => {
       <XYPlot width={300} height={300}>
         {Array(k)
           .fill(0)
-          .map(() => (
+          .map((_, index) => (
             <LineSeriesCanvas
+              key={index}
```
Comment (Xiot) [inline packages/react-vis/tests/components/line-series-canvas.test.js:16]: technically `key` should be a string.

File: packages/showcase/showcase-components/showcase-button.js [hunk 2]
```diff
@@ -30,7 +30,7 @@ function ShowcaseButton(props) {
   );
 }
 
-ShowcaseButton.PropTypes = {
+ShowcaseButton.ropTypes = {
```
Comment (Xiot) [inline packages/showcase/showcase-components/showcase-button.js:33]: `propTypes`

Comment (a-peltier) [reply packages/showcase/showcase-components/showcase-button.js:33]: nice catch.. thanks

File: packages/react-vis/tests/components/line-series-canvas.test.js (hunk 1 above)
Comment (a-peltier) [reply packages/react-vis/tests/components/line-series-canvas.test.js]: done

File: packages/showcase/showcase-components/showcase-button.js (hunk 2 above)
Comment (a-peltier) [reply packages/showcase/showcase-components/showcase-button.js]: done

**Pull request merged**